File: app/src/main/java/app/rental/utsavstha/rentalapp/Utils/ImageLoader.java
package app.rental.utsavstha.rentalapp.Utils;

import android.content.Context;
import android.widget.ImageView;

import com.squareup.picasso.Picasso;

/**
 * Created by utsavstha on 8/19/17.
 */

public class ImageLoader {

    public static void load(Context context,String url, ImageView imageView){
        Picasso.with(context).load(url).into(imageView);
    }
}

File: app/src/main/java/app/rental/utsavstha/rentalapp/Utils/ImageLoader.kt
package app.rental.utsavstha.rentalapp.Utils

import android.content.Context
import android.widget.ImageView

import com.squareup.picasso.Picasso

import app.rental.utsavstha.rentalapp.RentApplication

/**
 * Created by utsavstha on 8/19/17.
 */

object ImageLoader {

    fun load(url: String, imageView: ImageView) {
        Picasso.with(RentApplication.context).load(url).into(imageView)
    }
}
